wiki/Options/en.md

```diff
@@ -1,525 +1,162 @@
-</tr>
-<tr class="odd">
-<td><p>Use alternative font for chat display</p></td>
-<td><p><span class="plainlinks">Use old type font (<a href="https://en.wikipedia.org/wiki/Tahoma_%28typeface%29">Tahoma</a>) rather than current type font (<a href="https://www.daltonmaag.com/about/aller.html">Aller</a>) in <a href="Chat_Console" title="wikilink">Chat Console</a>. <a href="Media:OP_CHAT.jpg" title="wikilink">Comparison here</a>.</span></p></td>
-</tr>
-</tbody>
-</table>
+| Name                                  | Effect                                                                  |
+|---------------------------------------|-------------------------------------------------------------------------|
+| Select language                       | Select your preferred language.                                         |
+| Prefer metadata in original language  | Beatmaps will display their "original" untranslated metadata.           |
+| Use alternative font for chat display | Use old font (Tahoma) rather than current font (Aller) in Chat Console. |
 
 ### Updates
 
-<table style="width:36%;">
-<colgroup>
-<col width="12%" />
-<col width="24%" />
-</colgroup>
-<thead>
-<tr class="header">
-<th><p>Name</p></th>
-<th><p>Effect</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td><p>Release Stream</p></td>
-<td><p>1. <strong>Stable (Latest):</strong> Public build.</p>
-<hr>
-<p>2. <strong>Stable (Fallback):</strong> Public build (Older stable build for worst case scenario, default for fresh install)</p>
-<hr>
-<p>3. <strong>Beta:</strong> Like cutting edge, just more stable; new releases once a week</p>
-<hr>
-<p>4. <strong>Cutting Edge (Experimental):</strong> osu!test builds; released when feature is ready (<a href="Multi" class="uri" title="wikilink">Multi</a> is locked for non-supporters).</p>
-<hr>
-<p><strong>Note:</strong> Requires online connection to interchange between them.</p></td>
-</tr>
-<tr class="even">
-<td><p><em>Your osu! is up-to-date!</em></p></td>
-<td><p>Clicking it will forcibly check for a new version of osu!</p></td>
-</tr>
-<tr class="odd">
-<td><p>Open osu! folder</p></td>
-<td><p>Open osu! folder.</p></td>
-</tr>
-</tbody>
-</table>
-
-![](OP Graphics thumb.jpg "fig:OP Graphics thumb.jpg") Graphics section
------------------------------------------------------------------------
-
-<img src="optionsscreengraphics1.jpg" title="Graphics Tab" alt="Graphics Tab" width="300" />
+#### Release Stream
+
+| Name                        | Effect                                                         |
+|-----------------------------|----------------------------------------------------------------|
+| Stable (Latest)             | Public build.                                                  |
+| Stable (Fallback)           | Older public build in case any major bugs come up.             |
```

Comment:
**need a citation here** but I thought Fallback was the Stable (Lastest) build but used the original engine that osu! ran on or something (because Lastest uses OpenGL)
¯\\\_(ツ)\_/¯, but I wouldn't think it would be an old public build just because of a bug that broke Lastest